assets/prometheus-operator/deployment.yaml

```diff
@@ -20,9 +20,9 @@ spec:
         - --kubelet-service=kube-system/kubelet
         - --logtostderr=true
         - --config-reloader-image=quay.io/coreos/configmap-reload:v0.0.1
-        - --prometheus-config-reloader=quay.io/coreos/prometheus-config-reloader:v0.28.0
+        - --prometheus-config-reloader=quay.io/coreos/prometheus-config-reloader:v0.29.0
         - --namespaces=openshift-monitoring
-        image: quay.io/coreos/prometheus-operator:v0.28.0
+        image: quay.io/coreos/prometheus-operator:v0.29.0
```

Comment:
This pull request also updates the Prometheus Operator. Would you mind adding that to the commit message for future git bisecting? I guess changing the description of the pull request should include it in the merge commit?

Reply:
it doesn't actually, this is overridden at runtime